docs-src/docs/kognic-io/calibrations/cameras-custom.md

```diff
@@ -102,6 +102,10 @@ kognicutil wasm validate calibration.wasm
 
 ## Compilation
 
+:::caution
+Rust 1.82 removes support for multivalue target (returning multiple values from a function). This feature is need for the calibrations to work at the moment. Therefore, you need to pin your rust and/or cargo version to < 1.82.
```

Comment:
```suggestion
Rust 1.82.0 removes support for the multivalue feature target (returning multiple values from a function). Since this feature is currently needed for custom camera calibrations to work, the Rust and/or Cargo version needs to be pinned to < 1.82.0.
```